.circleci/config.yml

```diff
@@ -376,21 +376,21 @@ workflows:
           filters:
             branches:
               only: release
-      - ecr-build-and-push-image-staging:
-          requires:
-            - terraform-init-and-apply-to-staging
-          filters:
-            branches:
-              only: release
-      - terraform-deploy-ecs-task-to-staging:
-          requires:
-            - ecr-build-and-push-image-staging
-          filters:
-            branches:
-              only: release
+      # - ecr-build-and-push-image-staging:
+      #     requires:
+      #       - terraform-init-and-apply-to-staging
+      #     filters:
+      #       branches:
+      #         only: release
+      # - terraform-deploy-ecs-task-to-staging:
+      #     requires:
+      #       - ecr-build-and-push-image-staging
+      #     filters:
+      #       branches:
+      #         only: release
       - deploy-to-staging:
-          requires:
-            - terraform-deploy-ecs-task-to-staging
+          # requires:
+          #   - terraform-deploy-ecs-task-to-staging
```

Comment:
The `deploy-to-staging` should require `terraform-init-and-apply-to-staging`, otherwise it will trigger as a parallel workflow a the same time as `check-code-formatting` does. Also, it will fail, because it will not have workspace to attach to due to assume role not having run before it.